### PR TITLE
Update model and make tests compile

### PR DESCRIPTION
**Pull request by rocketninja7**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/EditCommandTest.java [hunk 1]
```diff
@@ -3,156 +3,94 @@
...
     @Test
-    public void execute_duplicatePersonUnfilteredList_failure() {
-        Person firstPerson = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
-        EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder(firstPerson).build();
-        EditCommand editCommand = new EditCommand(INDEX_SECOND_PERSON, descriptor);
-
-        assertCommandFailure(editCommand, model, EditCommand.MESSAGE_DUPLICATE_PERSON);
-    }
+    public void execute_noFieldSpecified_success() {
+        ModuleCode moduleCode = new ModuleCode(VALID_CODE_CS2040S);
```
Comment (xCOLOURx) [inline src/test/java/seedu/address/logic/commands/EditCommandTest.java:76]: Wait what happened to the } here, it is github being weird?

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java
Comment (xCOLOURx) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:1]: I think this test can be deleted and replaced with ModulePlanParserTest

File: src/test/java/seedu/address/model/AddressBookTest.java
Comment (xCOLOURx) [inline src/test/java/seedu/address/model/AddressBookTest.java:1]: Can be deleted too

**Review by xCOLOURx — COMMENTED** [review]
LGTM, just maybe can delete those 2 files also

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java
Comment (rocketninja7) [reply src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:1]: Will leave it to you to see how to do it

**Pull request merged**